pkg/sql/show.go

```diff
@@ -414,6 +414,20 @@ func (p *planner) showCreateTable(tn parser.Name, desc *sqlbase.TableDescriptor)
 	return buf.String(), nil
 }
 
+func makeIndexColNames(d sqlbase.IndexDescriptor) string {
+	var buf bytes.Buffer
+	for i, name := range d.ColumnNames {
+		if i > 0 {
+			buf.WriteString(", ")
+		}
+		buf.WriteString(parser.Name(name).String())
+		if dir := d.ColumnDirections[i]; dir != 0 {
```

Comment:
I would suggest to either

- drop the conditional on `dir != 0` and print the direction always, or
- print `ASC` always for direction 0.